src/mini-cast-bar/hooks/useMiniPlayerActionsHook.js

```diff
@@ -0,0 +1,36 @@
+import { VizbeeManager } from "../../../";
+import { ButtonType } from "../constants";
+import { LOG_TAG } from "../constants";
+
+export const useMiniPlayerActionsHook = () => {
```

Comment:
Can you add comments to explain the purpose of the hook and the main functions. Make sure you add the same for all the hooks.

Reply:
Done